oadp-operator/pkg/credentials/credentials.go

```diff
@@ -0,0 +1,85 @@
+package credentials
+
+import (
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type CloudProviderFields struct {
+	secretName         string
+	mountPath          string
+	envCredentialsFile string
+}
+
+const (
+	veleroSAName     = "velero"
```

Comment:
We are redeclaring these constants even though you use them in the other pkg. i would move these to a `common` pkg or something. Or import them from the other pkg